Packs/CyberArkPAS/pack_metadata.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "CyberArk",
+    "description": "-",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-06-03T21:49:36Z",
+    "categories": [
+        "Database"
```

not endpoint or authentication?
```suggestion
        "Endpoint"
```